### ISSUE TITLE
SDWebImageRetryFailed   How to use

### ISSUE THREAD

**Issue by 513494297** [rs/SDWebImage#1864]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

I have a question.When I close the wifi,and options is SDWebImageRetryFailed, the imageView not load the image.But when i open the wifi, the imageView still not load the image.So what is the SDWebImageRetryFailed meaning for.I'm new on the job。

**Comment by 513494297** [comment]
I have a question.When I close the wifi,and options is SDWebImageRetryFailed, the imageView not load the image.But when i open the wifi, the imageView still not load the image.So what is the SDWebImageRetryFailed meaning for.I'm new on the job。

**Comment by skyline75489** [comment]
It's clearly written in the documentation [here](https://github.com/rs/SDWebImage/blob/bd14ef3ac7a06e05ae74f6051f7b2a2034b26417/SDWebImage/SDWebImageManager.h#L19):

```objective-c
    /**
     * By default, when a URL fail to be downloaded, the URL is blacklisted so the library won't keep trying.
     * This flag disable this blacklisting.
     */
    SDWebImageRetryFailed = 1 << 0,
```

Please refer to any documentation you can possibly find before asking a question. This isn't about polite. Reading documentation is the very skill that you need to be a good coder.
